hail/python/test/hail/helpers.py

```diff
@@ -188,18 +161,7 @@ def wrapper(func, *args, **kwargs):
 
 
 def skip_unless_service_backend(reason='only relevant to service backend', clouds=None):
-    from hail.backend.service_backend import ServiceBackend
-
-    @decorator
-    def wrapper(func: Callable[P, T], *args: P.args, **kwargs: P.kwargs) -> T:
-        if not isinstance(hl.utils.java.Env.backend(), ServiceBackend):
-            raise unittest.SkipTest(reason)
-        else:
-            if clouds is None or os.environ['HAIL_CLOUD'] in clouds:
-                return func(*args, **kwargs)
-            raise unittest.SkipTest(f'{reason} for clouds {clouds}')
-
-    return wrapper
```

Comment:
What happened to the processing of the `clouds` parameter?

Reply:
oh good catch!